src/routes/+layout.svelte

```diff
@@ -27,4 +42,7 @@
 	});
 </script>
 
+{#if $authStore.user}
+	<button on:click={authHandlers.signout}>Sign Out from main layout</button>
```

Comment:
Let's change the button text here

Reply:
Thank you, I missed that. Removed the 'from main layout' part